client/components/createTM/tab-choose.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Col from 'react-bootstrap/lib/Col';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import { selectGame } from '../../actions/createTM-actions';
+
+class TabChoose extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange() {
+
+  const game=document.getElementById('formControlsSelect').value;
+  this.props.selectGame(game);
+  }
+  render() {
+      return (<div>
+        <Col sm={6}>
+          <h>Create your Tournament</h> <br />
+          <p>Instruction</p>
+        </Col>
+        <Col sm={6}>
+          <FormGroup controlId="formControlsSelect">
+            <ControlLabel>Select</ControlLabel>
+            <FormControl onChange={this.onChange} componentClass="select" placeholder="select">
+              <option value="Please Select" selected disabled>Please Selected--</option>
+              <option value="League of Legend">League of Legend</option>
```

Comment:
lets create a file: `/constants/games.js`

and define the game names just like action types. so we can avoid typo, and make game name value consistent throughout our code.

eg. `export const LeagueOfLegend = 'LeagueOfLegend';`
`export const HearthStone = 'HearthStone ';`